### DISCUSSION TITLE
[Solution] Adapter 'lldb-vscode' install on linux

### DISCUSSION THREAD

**Discussion by theCollectiv** [mfussenegger/nvim-dap#521] [Show and tell]
I had quite a hard time, to get the right binary `lldb-vscode[...]` installed for linux. But It's really easy, so please **add** this to the **nvim-dap documentation** (like its done here, but for linux: [Debug Adapter installation · mfussenegger/nvim-dap Wiki](https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation#installation-on-mac) ).

There are hardly any search results on google for "rust lldb-vscode". So here is the **solution**:
It's not the `llvm`package, but the `lldb` package, which contains the `lldb-vscode[-version]` binary.

On Fedora its `sudo dnf -y install lldb`.
On Ubuntu its `sudo install lldb`.

After the installation, its in `usr/bin/lldb-vscode-10` for example.

[How To Install lldb on Fedora 34 | Installati.one](https://installati.one/fedora/34/lldb/)
[Ubuntu – Details of package lldb in focal](https://packages.ubuntu.com/focal/lldb)
[Ubuntu – File list of package lldb-10/focal/amd64](https://packages.ubuntu.com/focal/amd64/lldb-10/filelist)

**Comment by mfussenegger** [comment]
The wiki is open for anyone to edit.

Also a tip: `dnf provides` can be used to find which package provides an executable.

For example:

```
# dnf provides lldb-vscode
Last metadata expiration check: 2:11:38 ago on Sun Apr 17 14:06:25 2022.
lldb-12.0.1-1.module_el8.5.0+892+54d791e1.i686 : Next generation high-performance debugger
Repo        : appstream
Matched from:
Filename    : /usr/bin/lldb-vscode
```
